libcxx/test/libcxx/vendor/ibm/bad_function_call.pass.cpp

```diff
@@ -6,7 +6,7 @@
 //
 //===----------------------------------------------------------------------===//
 
-// REQUIRES: target={{powerpc.*-ibm-aix.*}}
+// REQUIRES: target={{.+}}-aix{{.*}}
```

Comment:
Why remove the vendor in the vendor specific directory?

Reply:
> Why remove the vendor in the vendor specific directory?

IBM is the only vendor shipping the AIX OS so I think the vendor part is unnecessary. This is similar to `{{.+}}-zos{{.*}}`.

Reply:
But then we should be able to hoist these tests out of the vendor/ibm directory and into the general test set? Perhaps @ldionne has opinions on this.

Reply:
I see what you mean, thanks! I can do it if we think it is better to move them out of the `vendor/ibm` directory.

Reply:
`libcxx/test/vendor` is a bit mis-named, I guess. It's more like `libcxx/test/vendor-or-extremely-platform-specific-stuff`. I think it makes more sense to keep this here since this test doesn't make sense outside of AIX.

Reply:
Thanks!